src/hospital/PatientCollection.java

```diff
@@ -0,0 +1,46 @@
+package hospital;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ *
+ * @author maximdumont
+ */
+public class PatientCollection{
+    private List<Patient> patients = new ArrayList<>();
+    
+    public PatientCollection(){
+        
+    }
+    
+    public PatientCollection(List<Patient> patients){
+        this.patients = this.patients;
+    }
+    
+    public void add(Patient p){
+        patients.add(p);
+    }
+    
+    public void add(Patient[] patients){
+        for(Patient p : patients){
+            this.patients.add(p);
+        }
+    }
+    
+    public void remove(Patient p){
+        patients.remove(p);
+    }
+    
+    public void remove(Patient[] p){
+        for(Patient patient:p){
+            patients.remove(p);
+        }
+    }
+    
+    public boolean contains(Patient p){
```

Comment:
This may not be feasible to use since the caller would already need to have a reference to the Patient object. If we support lookups by patient ID then we can just check for a key in the HashMap or whatever is appropriate for the implementation.